packages/gestalt/src/RadioGroupButton.tsx

```diff
@@ -130,27 +140,78 @@ const RadioGroupButtonWithForwardRef = forwardRef<HTMLInputElement, Props>(funct
     borderWidth = styles.BorderCheckedMd;
   }
 
+  const uncheckedBorderWidth = disabled ? styles.BorderDisabled : styles.BorderUnchecked;
+
   const styleSize = size === 'sm' ? controlStyles.sizeSm : controlStyles.sizeMd;
 
   const bgStyle = disabled && !checked ? styles.BgDisabled : styles.BgEnabled;
 
   const { isFocusVisible } = useFocusVisible();
 
   const { parentName } = useRadioGroupContext();
+
+  const tapScaleAnimation = useTapScaleAnimation();
```

Comment:
nit  

 const { handleMouseDown, handleMouseUp } = useTapScaleAnimation();

Reply:
I didn't destructure the object to keeps its context "visible" while reading the code. We may have other even handler functions in the future. But this one is just for the animation. That's why i think its better to keep it this way.